mteb/tasks/Reranking/multilingual/MIRACLReranking.py

```diff
@@ -4,35 +4,88 @@
+    issn = {2307-387X},
+    doi = {10.1162/tacl_a_00595},
+    url = {https://doi.org/10.1162/tacl\_a\_00595},
+    eprint = {https://direct.mit.edu/tacl/article-pdf/doi/10.1162/tacl\_a\_00595/2157340/tacl\_a\_00595.pdf},
+}"""
 
 
 class MIRACLReranking(MultilingualTask, AbsTaskReranking):
     metadata = TaskMetadata(
         name="MIRACLReranking",
-        description="MIRACL (Multilingual Information Retrieval Across a Continuum of Languages) is a multilingual retrieval dataset that focuses on search across 18 different languages. This task focuses on the German and Spanish subset.",
+        description="MIRACL (Multilingual Information Retrieval Across a Continuum of Languages) is a multilingual retrieval dataset that focuses on search across 18 different languages.",
         reference="https://project-miracl.github.io/",
         dataset={
-            "path": "jinaai/miracl",
-            "revision": "d28a029f35c4ff7f616df47b0edf54e6882395e6",
+            "path": "miracl/mmteb-miracl-reranking",
+            "revision": "6d1962c527217f8927fca80f890f14f36b2802af",
         },
         type="Reranking",
         category="s2s",
-        eval_splits=["test"],
-        eval_langs={
-            "de": ["deu-Latn"],
-            "es": ["spa-Latn"],
-        },
-        main_score="map",
-        date=None,
-        form=None,
-        domains=None,
-        task_subtypes=None,
-        license=None,
-        socioeconomic_status=None,
-        annotations_creators=None,
-        dialect=None,
-        text_creation=None,
-        bibtex_citation=None,
+        eval_splits=[_EVAL_SPLIT],
+        eval_langs=_LANGUAGES,
+        main_score="NDCG@10",
+        date=("2022-06-01", "2023-01-30"),
+        form=["written"],
+        domains=["Encyclopaedic"],
+        task_subtypes=[],
+        license="CC BY-SA 4.0",
+        socioeconomic_status="low",
```

Comment:
```suggestion
        socioeconomic_status="mixed",
```
as the documents are from wikipedia